client/src/cordova/apple/xcode/Outline/Classes/AppKitBridge/Resources/Strings/en.lproj/Localizable.strings

```diff
@@ -2,3 +2,5 @@
 "disconnected_server_state" = "Disconnected";
 "tray_open_window" = "Open";
 "quit" = "Quit";
+"disconnect" = "Disconnect";
```

Comment:
*open question*

We can merge these messages if they function correctly on non-English macOS. Otherwise, translation might be necessary.

Reply:
I can remove the `Connected` and `Disconnected` but new translations are still needed without it sounding weird. 

For example in French, `Disconnect` is `Déconnecter` and `Disconnected` is `Déconnecté`

Reply:
Weird appearance is better than crashing, I can accept showing English everywhere.

Reply:
Is there anything you would like me to do on my end?